source/packages-software-libraries.rst

```diff
@@ -180,17 +191,41 @@ Popular Linux System Package Managers
 .. ifnotslides::
 
     These are package managers used to install system packages like
-    Web-browsers, terminals, netowrk managers, etc.  Although they are
+    Web-browsers, terminals, network managers, etc.  Although they are
     default OS package managers, a package manager itself is really just a
     package itself, so one can install *any* package manager they want.
 
-Popular Linux Package Managers:
+**.deb**
+
+* apt - Debian package manager with repo support
+* dpkg - low level package manager tool used by apt
+* Used by Debian, Ubuntu, Linux Mint and others
+
+**.rpm**
+
+* yum - RPM Package manager with repo support
+* rpm - low level package manager tool used by yum
+* Used by RedHat, CentOS, Fedora and others
+
+Yum vs. Apt
+-----------
+
+**Yum**
 
-Apt (``.deb``, ``dpkg``)
-    Used by default on the Debian, Ubuntu, Linux Mint operating systems.
+* XML repository format
+* Automatic metadata syncing
+* Supports a plugin module system to make it extensible
+* Checks all dependencies before downloading
+
+**Apt**
+
+* Upgrade and Dist-Upgrade
+
+  * Dist-Upgrade applies intelligent upgrading decisions during a major system
+    upgrade
+
+* Can completely remove all files including config files
```

Comment:
ordering goes from `deb` -> `rpm` above, then reverses to `yum` -> `apt` here, potentially confusing